schemas.py

```diff
@@ -0,0 +1,38 @@
+from datetime import date
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: Optional[str] = None
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class Author(AuthorBase):
+    id: int
+
+    class Config:
+        from_attributes = True
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+
+class BookCreate(BookBase):
+    author_id: int
+
+
+class Book(BookBase):
+    id: int
+    author: Author
+
+    class Config:
+        from_attributes = True
```

Comment:
The `from_attributes` configuration in the `Book` model's `Config` class is not a standard Pydantic configuration option. Ensure this is intended or remove it if not needed.